lib/transformations/resolve/resolve.js

```diff
@@ -0,0 +1,46 @@
+// Press ctrl+space for code completion
+export default function transformer(file, api) {
+	const j = api.jscodeshift;
+  
```

Comment:
In general, I am more a fan of using whole words instead of letters or abbreviations for variables and/or functions, it helps new developers to have a better understanding on the code they are reading